controllers/devworkspace/solver/che_routing.go

```diff
@@ -514,13 +514,10 @@ func addEndpointToTraefikConfig(componentName string, e dw.Endpoint, cfg *gatewa
 		100,
 		fmt.Sprintf("http://127.0.0.1:%d", e.TargetPort),
 		[]string{prefix})
-	if util.IsOpenShift4 {
-		cfg.AddAuth(name, fmt.Sprintf("http://%s.%s:8089?namespace=%s", gateway.GatewayServiceName, cheCluster.Namespace, routing.Namespace))
-	}
+	cfg.AddAuth(name, fmt.Sprintf("http://%s.%s:8089?namespace=%s", gateway.GatewayServiceName, cheCluster.Namespace, routing.Namespace))
```

Comment:
This reminds me... Are we still meant to support OpenShift 3? If yes, how is this going to behave on OpenShift 3?

Reply:
TBH I don't know. But looking at current `che_routing.go`, I doubt it's working now.